dj/api/helpers.py

```diff
@@ -125,53 +111,41 @@ def get_catalog(session: Session, name: str) -> Catalog:
     return catalog
 
 
-async def get_query(  # pylint: disable=too-many-arguments
+def get_query(  # pylint: disable=too-many-arguments
     session: Session,
     metric: str,
     dimensions: List[str],
     filters: List[str],
-    database_name: Optional[str] = None,
-    check_database_online: bool = True,
-) -> Tuple[ast.Query, Database]:
+) -> ast.Query:
     """
     Get a query for a metric, dimensions, and filters
     """
     metric = get_node_by_name(session=session, name=metric, node_type=NodeType.METRIC)
-    database_id = (
-        get_database_by_name(session=session, name=database_name).id
-        if database_name
-        else None
-    )
-    query_ast, optimal_database = await build_node_for_database(
+
+    query_ast = build_node(
```

Comment:
Does this need to also include the engine? As in, now instead of building the query for a database, we're building the query for an engine. Or is the idea that this is just a generic query AST, which would then need to be translated to engine-specific SQL by the query engine?